test_gpios.py

```diff
@@ -20,9 +20,26 @@ async def asyncSetUp(self):
         self.robot = await RobotClient.at_address(conf.address, opts)
         board = Board.from_robot(self.robot, "board")
         self.input_pin = await board.gpio_pin_by_name(conf.INPUT_PIN)
-        self.interrupt = await board.digital_interrupt_by_name(conf.INPUT_PIN)
         self.output_pin = await board.gpio_pin_by_name(conf.OUTPUT_PIN)
 
+        # Most boards have combination GPIO/PWM/interrupt pins. However, rarely
+        # they are separated to different pins (e.g., the Beaglebone AI-64 does
+        # not have GPIO functionality on the PWM pins or vice versa). If you
+        # define them separately, define these extra variables in the config.
+        # Otherwise, we'll reuse the same inputs and outputs.
```

Comment:
Ugh. The last two sentences don't make as much sense as I thought on Friday. I'll rephrase them.